lib/provisioner/worker/plugins/automators/docker_automator/docker_automator.json

```diff
@@ -16,9 +16,15 @@
           },
           "publish_ports": {
             "label": "Container ports to publish",
-            "override": false,
+            "override": true,
             "type": "text",
             "tip": "Comma-separated list of container ports to publish"
+          },
+          "command": {
+            "label": "Command to run, including arguments",
```

Comment:
``command`` is misleading to me.  it's not clear that it shouldn't include the image or the port mapping arguments, so it's not actually the "Command to run".  I believe you have this exact error in https://github.com/caskdata/coopr-templates/pull/75/files (image name repeated).  ``additional_args`` or similar would be better IMO.